src/main/java/io/jenkins/plugins/coverage/threshold/Threshold.java

```diff
@@ -1,49 +1,85 @@
 package io.jenkins.plugins.coverage.threshold;
 
+import hudson.Extension;
 import hudson.ExtensionPoint;
 import hudson.model.Describable;
 import hudson.model.Descriptor;
 import io.jenkins.plugins.coverage.targets.CoverageMetric;
 import jenkins.model.Jenkins;
 import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.util.Objects;
 
 public class Threshold implements ExtensionPoint, Describable<Threshold> {
+
     private final CoverageMetric threshTarget;
 
-    private final float healthyThresh;
-    private final float unstableThresh;
-    private final float unhealthyThresh;
+    // mark build as unstable when coverage is less than this
+    private float unstableThresh = 0.0f;
```

Comment:
as suggested in parallel PR, better to use "threshold" instead of "thresh" everywhere